data/nested_cubes.msh

```diff
@@ -0,0 +1,2300 @@
+$MeshFormat
```

Comment:
It seems this mesh is added, because the inner cube is difficult to generate and cannot be done simply with code such as `MakeCartesian3D`. If it is necessary to add such a mesh, is it possible to make a coarser version, to keep the file added to the repo of minimal size? If the tests should use a finer mesh, it would be preferable to generate the coarsest mesh possible and refine it within the code.